### PR TITLE
Updating jumbo-jekyll-theme version to 5.5.1

### PR DESCRIPTION
**Pull request by kylekirkby**
Updating jumbo-jekyll-theme version to 5.5.1

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+build-site.sh   @pcolmer @pcolmer-linaro @morancj
```
Comment (morancj) [inline CODEOWNERS]:
```suggestion
build-site.sh   @pcolmer @morancj
```
Old GitHub account

File: _data/sticky-tab-bar.yml [hunk 2]
```diff
@@ -13,12 +13,3 @@ pages:
...
     urls: [/governance/,/releases/,/committee/]
-  -
-    list:
-      -
-        title: Documentation
-        url: /documentation/
-      -
-        title: License Agreement
-        url: /agreement/
-    urls: [/documentation/, /agreement/]
```
Comment (morancj) [inline _data/sticky-tab-bar.yml]: Missing line feed

Comment (morancj) [inline _data/sticky-tab-bar.yml]:
```suggestion
    urls: [/documentation/, /agreement/]

```

**Comment by kylekirkby** [comment]
@pcolmer @morancj - is there anything I need to fix before this can merged ? BR, Kyle

**Comment by pcolmer** [comment]
Looks like @morancj has to change his review from requested changes to approved :)

**Pull request merged**